### DISCUSSION TITLE
Wrong alpine version in proxmox?

### DISCUSSION THREAD

**Discussion by WilJ541** [ej52/proxmox-scripts#123] [Q&A]
in proxmox 8 from the shell if you:
 curl -sL https://raw.githubusercontent.com/ej52/proxmox/main/lxc/nginx-proxy-manager/create.sh | bash -s
you get an error:
[error] No LXC template found for alpine-3.16
however if you look, there's a template called alpine-3.16-default_20220622_amd64.tar.gz
i downloaded it and manually setup a container then ran:
wget --no-cache -qO - https://raw.githubusercontent.com/ej52/proxmox/main/lxc/nginx-proxy-manager/setup.sh | sh

everything appeared to work fine, until i try to add a LetsEncrypt SSL.  no matter how i choose to add the cert i get this error:
Error: Command failed: . /opt/certbot/bin/activate && pip install --no-cache-dir --user certbot-dns-namecheap~=1.0.0 && deactivate
ERROR: Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
[notice] A new release of pip is available: 23.0.1 -> 23.1.2
[notice] To update, run: pip install --upgrade pip
at ChildProcess.exithandler (node:child_process:402:12)
at ChildProcess.emit (node:events:513:28)
at maybeClose (node:internal/child_process:1100:16)
at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)

it's a lab, if you can help i'd appreciate it.
reading thru all this again to proofread and i notice the error says NoCacheDir.  so does the installer command i used.  interesting

**Comment by WilJ541** [comment]
Bummer.   I hoped that you as the dev would take enough pride in your project to help.
but all the empty discussions show otherwise, so i guess i just added another one

too bad, i liked NPM.

**Reply by lludlow** [reply]
go and grab an alpine 3.16 image and add it to proxmox. It looks like proxmox only offers 3.18 by default. and FYI this is just some guy that made an install script that unofficially installs this on proxmox / LXC. The officially supported way would be to use docker https://github.com/NginxProxyManager/nginx-proxy-manager

If you need the latest alpine 3.16 image you can grab it from https://images.linuxcontainers.org/

**Comment by WilJ541** [comment]
i have 3.16, thnx, that's not the issue

**Reply by lludlow** [reply]
I missed that.

Go to the pyvenv.cfg file in the Virtual environment folder (I think it is /opt/letsencrypt/

Set the include-system-site-packages to true and save the change
